src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -106,6 +106,17 @@ protected override void OnPersist(ApplicationEngine engine)
         {
             base.OnPersist(engine);
 
+            // Set next validators
+
+            var committee = GetCommitteeMembers(engine.Snapshot).ToArray();
+            StorageItem storage = engine.Snapshot.Storages.GetAndChange(CreateStorageKey(Prefix_NextValidators), () => new StorageItem());
+            storage.Value = committee.Take(ProtocolSettings.Default.ValidatorsCount).OrderBy(p => p).ToArray().ToByteArray();
```

Comment:
Move these lines to `PostPersist()` too? Otherwise the committee could be different from `PostPersist()`.